src/components/pages/RepoPage.js

```diff
@@ -89,11 +89,14 @@ export const RepoPage = ({ match }) => {
     <div>
       <SwitchUserButton />
       <Heading fontSize="28px" mt="25px" textAlign="left">
-        <Link href={repoURL} isExternal>
+        <Link href={repoURL} textDecoration="underline" isExternal>
           {repo}
         </Link>
-        <Text style={{ display: 'inline' }}> by </Text>
-        <Link href={userURL} isExternal>
+        <Text color="#808080" style={{ display: 'inline' }}>
+          {' '}
+          by{' '}
+        </Text>
+        <Link href={userURL} textDecoration="underline" isExternal>
```

Comment:
I noticed on the user page, we have some text that says  user's repositories. can we add some text on the page that says commit history or something like that?